Example/AblyChatExample/Mocks/MockRealtime.swift

```diff
@@ -10,6 +11,168 @@ final class MockRealtime: NSObject, ARTRealtimeProtocol, Sendable {
         fatalError("Not implemented")
     }
 
+    let channels = Channels()
+
+    final class Channels: RealtimeChannelsProtocol {
+        final class Channel: RealtimeChannelProtocol {
```

Comment:
Is there any benefit to make them nested?

Reply:
No, didn't really give it any thought. But less intentation is nice I guess, so will un-nest.